tests/functional/adapter/conftest.py

```diff
@@ -1,4 +1,36 @@
 import pytest
+import os
```

Comment:
@soksamnanglim added this to make sure the grants/roles get created on the test DB